tests/test_PR.py

```diff
@@ -112,5 +113,18 @@ def test_PR_529():
     assert video_clip.rotation == 180
 
 
+def test_PR_610():
```

Comment:
I wonder if we should  have a better name and file name. Something more descriptive. In the future no one will remember what PR 610 was. Better to name if after what is also tested. Maybe composite_video_clip_max_fps?